.github/workflows/elixir_apps_check.yaml

```diff
@@ -35,9 +35,9 @@ jobs:
         run: ./scripts/check-elixir-applications.exs
         env:
           MIX_ENV: ${{ matrix.profile }}
-      - name: check applications started with emqx_machine
-        run: ./scripts/check-elixir-emqx-machine-boot-discrepancies.exs
-        env:
-          MIX_ENV: ${{ matrix.profile }}
+#      - name: check applications started with emqx_machine
```

Comment:
This will possibly make `mix.exs` and `rebar.config` diverge silently, since the order and presence of applications is important for both releases.